content/blog/2017/2017-01-17-Jenkins-is-upgrading-to-Java-8.adoc

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Jenkins Upgrades To Java 8"
+tags:
+- java8
+- upgrade
+author: batmat
+---
+
+NOTE: Back in last November, we discussed interesting statistics showing that Jenkins was now running Java 8 on a majority of its running instances.
+This was _52.8%_ back then, and now reaching link:http://stats.jenkins.io/plugin-installation-trend/jvms.json[_58%_ two months later].
+If we only look at Jenkins 2.x, then we reach _72%_.
+
+== Upgrading to Java 8
+
+Balancing those numbers with many other criteria:
+
+* Java 7 has been now end-of-lifed for 18+ months ;
+* People are already moving away from Java 7, as show the numbers ;
+* Java 8 runtime is known from the field to be more stable
+** Many developers have been wanting to be allowed to leverage the improvements that Java 8 provides to the language and platform
```

Comment:
`s/**/*/`
When doing bullets don't make lone bullet.

`s/wanting to be allowed to leverage/wanting to leverage/`